packages/design-system/src/components/UsaBanner/UsaBanner.tsx

```diff
@@ -28,6 +28,25 @@ export interface UsaBannerProps {
  * [refer to its full documentation page](https://design.cms.gov/components/usa-banner/).
  */
 export const UsaBanner: React.FunctionComponent<UsaBannerProps> = (props: UsaBannerProps) => {
+  // Logic to see deprecated CSS in use
+  useEffect(() => {
+    window.addEventListener('load', handleRefresh);
+
+    return () => {
+      window.removeEventListener('load', handleRefresh);
+    };
```

Comment:
I'm testing this out in storybook, and I can't get the `handleRefresh` function to execute. I think the window `load` event is firing long before we get here. I don't think we need these at all. I think you should be able to put the contents of `handleRefresh` directly in this useEffect

Reply:
Also, in order for us not to lose it, and so we can better test it, we should probably put this into its own hook in the utilities folder like
```js
function useCssDeprecationDebugger() {
  useEffect(...)
}
```

Reply:
If we do that, we might be able to write a unit test with jsdom where we define the `--debugger` variable and verify that it behaves differently when it's pre-defined